docs/explanations/faq.md

```diff
@@ -270,7 +270,7 @@ This is the canonical list of keyboard shortcuts:
 		</tr>
 		<tr>
 			<td>Remove multiple selected blocks.</td>
-			<td></td>
```

Comment:
We have exactly the same info bellow. Could we just delete the `td`?

Reply:
The table felt in-complete. Although not an issue or anything. 
![image](https://user-images.githubusercontent.com/33965848/210063569-48f0b9a2-09e7-499c-92b6-b4836be04d2b.png)

Reply:
Thank you 👍